environments/icds-cas/public.yml

```diff
@@ -27,7 +27,8 @@ swift_url: https://mowcdmum.ipstorage.tatacommunications.com/auth/v1.0/
 
 nofile_limit: 65536
 
-redis_maxmemory: 6gb
+redis_maxmemory: 8gb
```

Comment:
I'm concerned that this is going to bite us in the future. See https://github.com/dimagi/commcare-cloud/pull/2929#issuecomment-499970466

Reply:
I think we have 16GB machines so, 8 should be fine even with that, since it's half? Also, I will look a bit more to see if we can avoid the rewrite to happen using peak usage so that we can get away with limitation.